CONTRIBUTING.md

```diff
@@ -0,0 +1,114 @@
+# Contributing guidelines
+
+Welcome to [TheAlgorithms/Nim](https://github.com/TheAlgorithms/Nim)! Before sending your pull requests, make sure that you __read the whole guidelines__. If you have any doubt on the contributing guide, please feel free to [state it clearly in an issue](https://github.com/TheAlgorithms/Nim/issues/new) or ask the community in [Discord](https://discord.gg/c7MnfGFGa6).
+
+## Contributing
+
+We are very happy that you are considering implementing algorithms and data structures for others!
+
+ This repository is referenced and used by learners from all over the globe. Being one of our contributors, you agree and confirm that:
```

Comment:
```suggestion
We are very happy that you are considering implementing algorithms and data structures for others!\
This repository is referenced and used by learners from all over the globe. Being one of our contributors, you agree and confirm that:
```

This should look something similar to the following (`\` stands for a newline).

---

We are very happy that you are considering implementing algorithms and data structures for others!\
This repository is referenced and used by learners from all over the globe. Being one of our contributors, you agree and confirm that:

Reply:
I commit this, but I prefer blank spaces than visible characters that are escaped.
People may read the CONTRIBUTING.md in CLI directly rather than in GitHub.